app/bundles/LeadBundle/Assets/css/lead.css

```diff
@@ -206,6 +206,37 @@ ul.tag-cloud li {
 .inline-spacer {
     text-transform: capitalize;
 }
+.contact{
+    display: inline !important;
+}
+.channel-disabled{
+    color:lightgrey;
+}
+.channel-enabled{
+    color:#47535f;
+}
+.frequency-table, .frequency-table td{
+    border-bottom: none!important;
+    border-top: none!important;
+}
+
+.frequency-values{
+    padding-left: 10px;
+    padding-top: 27px;
+    padding-bottom: 15px;
+}
+.frequency-select{
+    width:120px!important;
+    float: right;
+    padding-right: 5px;
+}
+.frequency-label{
+     padding: 8px;
```

Comment:
It seems that CS Fixer doesn't fix CSS styles. To have the code style the same as the rest of the file, could you remove the additional space here (4 instead of 5) and add the spaces between the `.[class-name]` and `{`?